.agents/rules/commit.md

```
@@ -0,0 +1,21 @@
+# Commit Rules
+
+Follow [Conventional Commits](https://www.conventionalcommits.org) for the commit message. Write commit messages in English.
+
+**Format:** `<type>(<scope>): <description>`
+
+- `scope` is the package name, component name, or area of change (e.g., `button`, `utils`, `docs`, `deps`, `changesets`).
+- `description` starts with a lowercase verb.
+
+**Examples:**
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)